pyproject.toml

```diff
@@ -87,7 +91,7 @@ Jinja2 = "~2.11.3"
 # Optional data validation of config contexts
 jsonschema = "~3.2.0"
 # Rendering of markdown files to HTML
-Markdown = "~3.3.4"
+Markdown = "3.3.4"
```

Comment:
Looks like the issue with 3.3.5 has been fixed in 3.3.6. Can we make this `~3.3.6` instead of pinning to 3.3.4 exactly?